### PR TITLE
calculator

### PR DESCRIPTION
**Pull request by shivamvashishth**
CRMON-ABC (TEST-NOTIFICATIONS)

### CODE REVIEW

File: core-java-module/src/main/java/programs/math/calculator.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package programs.math;
+
+public class calculator {
```
Comment (ankitrawt93) [inline core-java-module/src/main/java/programs/math/calculator.java:3]: Class name should start with capital letters

Comment (KshitijBaluni) [inline core-java-module/src/main/java/programs/math/calculator.java:3]: class java docs missing.

File: core-java-module/src/main/java/programs/math/calculator.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    private int x;
+    private int y;
+    private int z;
+
+
+    public  calculator(int x, int y){
```
Comment (KshitijBaluni) [inline core-java-module/src/main/java/programs/math/calculator.java:9]: Class name should be Calculator

File: core-java-module/src/main/java/programs/math/calculator.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        this.x=x;
+        this.y=y;
+    }
+
```
Comment (KshitijBaluni) [inline core-java-module/src/main/java/programs/math/calculator.java:13]: add doc for every function.

**Review by KshitijBaluni — COMMENTED** [review]
Write unit test for the class and resolve the following comments.

File: core-java-module/src/main/java/programs/math/calculator.java (hunk 1 above)
Comment (shivamvashishth) [reply core-java-module/src/main/java/programs/math/calculator.java:3]: resolved

**Pull request closed without merging**